pkgs/tools/misc/diffoscope/default.nix

```
@@ -3,7 +3,8 @@
 , e2fsprogs, enjarify, file, findutils, fontforge-fonttools, ffmpeg, fpc, gettext, ghc, ghostscriptX, giflib, gnumeric, gnupg, gnutar
 , gzip, html2text, hdf5, imagemagick, jdk, libarchive, libcaca, llvm, lz4, mono, ocaml, oggvideotools, openssh, openssl, pdftk, pgpdump, poppler_utils, procyon, qemu, R
 , radare2, sng, sqlite, squashfsTools, tcpdump, ubootTools, odt2txt, unzip, wabt, xmlbeans, xxd, xz, zip, zstd
-, enableBloat ? false
+, enableBloat ? !stdenv.isDarwin
```

```suggestion
, enableBloat ? true
```
if minimal sets this to false, it is less confusing if we default this to true.

Also I don't think it makes sense to always get the diffoscopeMinimal on darwin and on linux you get the full one with diffoscope. That's almost always unexpected. If packages don't work on darwin then they should be rather moved to linux only like it is already done for some.

---

cc @alyssais who suggested to me this change.

---

> Also I don't think it makes sense to always get the diffoscopeMinimal on darwin and on linux you get the full one with diffoscope. That's almost always unexpected. If packages don't work on darwin then they should be rather moved to linux only like it is already done for some.

This makes sense to me.

---

Let's see what breaks on Darwin and let's move it then